### ISSUE TITLE
I did not see a video support.

### ISSUE THREAD

**Issue by surajpaudelNEPAL** [tjcampanella/story_time#9]
Some stories can be the video so is it possible for me to create a video widget with the video_player package and use this widget inside  StoryViewPage as StoryImage? Does it lead to any problems?

**Comment by bb-renittojose** [comment]
@surajpaudelNEPAL Yes you can use video_player package for video support on the Video level. i have a similar implementation to support following media types - image, gif and video

```dart
class StoryMediaWidget extends StatelessWidget {
.
.
// your custom logic
.
Widget build(BuildContext context) {
.
// your custom logic
.
 if (storyMediaStateData.isImage()) {
                return CacheNetworkImage(
                  imageUrl: slide?.media?.image ?? "",
                );
              } else if (storyMediaStateData.isVideo()) {
                return VideoWidget(
                  mediaStateData: storyMediaStateData,
                  sizeConfig: sizeConfig,
                  height: 300,
                  width: 300,
                );
              } else if (storyMediaStateData.isGif()) {
                return CacheNetworkImage(imageUrl: slide?.media?.gif ?? "");
              }
}
}
``` 

@surajpaudelNEPAL  does this package support 'StoryImage'?

**Comment by sarbazx** [comment]
> @surajpaudelNEPAL Yes you can use video_player package for video support on the Video level. i have a similar implementation to support following media types - image, gif and video
> 
> ```dart
> class StoryMediaWidget extends StatelessWidget {
> .
> .
> // your custom logic
> .
> Widget build(BuildContext context) {
> .
> // your custom logic
> .
>  if (storyMediaStateData.isImage()) {
>                 return CacheNetworkImage(
>                   imageUrl: slide?.media?.image ?? "",
>                 );
>               } else if (storyMediaStateData.isVideo()) {
>                 return VideoWidget(
>                   mediaStateData: storyMediaStateData,
>                   sizeConfig: sizeConfig,
>                   height: 300,
>                   width: 300,
>                 );
>               } else if (storyMediaStateData.isGif()) {
>                 return CacheNetworkImage(imageUrl: slide?.media?.gif ?? "");
>               }
> }
> }
> ```
> 
> @surajpaudelNEPAL does this package support 'StoryImage'?

how do you pause video when story is on hold?
